Sync sample permission mappings config

phuongnq opened the pull request:
https://github.com/craftercms/craftercms/issues/7712

phuongnq committed:
Sync sample permission mappings config

alhambrav reviewed (changes requested):
Please remove the following permissions that does not exist anymore in the sample-permission-mappings.xml file:

- change_content_type
- publish_clear_lock
- rebuild_database

phuongnq committed:
Remove unused permissions

phuongnq commented:
> Please remove the following permissions that does not exist anymore in the sample-permission-mappings.xml file:
> 
> * change_content_type
> * publish_clear_lock
> * rebuild_database

I updated this to remove the above permissions. I also updated the internal BPs, and global permission mappings file as well. 
A ticket is created to track this PR. cc @sumerjabri

alhambrav reviewed (changes requested):
Hi! There are a few more permissions that are not used anymore:

- create_cluster
- delete_cluster
- update_cluster
- read_logs

phuongnq committed:
Remove unused cluster permissions

phuongnq commented:
@sumerjabri @alhambrav I removed the following permissions:
```
delete_cluster
update_cluster
```

The others are still in use for the global menu. For example: https://github.com/craftercms/studio/blob/develop/src/main/webapp/repo-bootstrap/global/configuration/global-menu-config.xml#L67

sumerjabri commented:
This was done by https://github.com/craftercms/studio/pull/3693 as part of the publishing refactor.
